File: CS/2011_Final_Value_of_Variable_After_Performing_Operations.cs
public class Solution {
    public int FinalValueAfterOperations(string[] operations) {
        int x = 0;
        for(int i = 0; i < operations.Length; i++){
            if(operations[i].Equals("++X") || operations[i].Equals("X++")){
                x++;
            }
            else{
                x--;
            }
        }
        return x;
    }
}

File: Python3/2011_Final_Value_of_Variable_After_Performing_Operations.py
class Solution(object):
    def finalValueAfterOperations(self, operations):
        # return sum (1 if '+' in operation else -1 for operation in operations)
        result = 0
        for op in operations:
            if op in ["--X", "X--"]:
            #if op == "--X" or op == "X--":
                result += -1
            else:
                result += 1
        return result
        """
        :type operations: List[str]
        :rtype: int
        """
        
